source/connect/connection-targets.txt

```diff
@@ -0,0 +1,116 @@
+.. _php-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client, stable api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and ``MongoDB\Client`` object
+to connect to different types of MongoDB deployments.
+
+.. _php-connection-atlas:
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- URI of your Atlas cluster
+- MongoDB username
+- MongoDB password
+
+Then, pass your connection string to the ``MongoDB\Client`` constructor.
+
+When you connect to Atlas, we recommend using the {+stable-api+} client option to avoid
+breaking changes when Atlas upgrades to a new version of {+mdb-server+}.
+To learn more about the {+stable-api+} feature, see the :ref:`{+stable-api+} page
+<php-stable-api>`.
+
+The following code shows how to use the {+driver-short+} to connect to an Atlas cluster.
+The code also uses the ``serverApi`` option to specify a {+stable-api+} version.
+
+.. literalinclude:: /includes/connect/atlas.php
+   :copyable: true
+   :language: php
+
+.. tip::
+
+   Follow the :atlas:`Atlas driver connection guide </driver-connection>`
```

Comment:
S: you can link to the Connection String step of the quick start here instead
```suggestion
   Follow the :ref:`php-connection-string` step of the Quick Start
```

Reply:
good idea!